docs/build/cmake-presets-vs.md

```diff
@@ -223,6 +223,22 @@ Build with `clang`:
 }
 ```
 
+*OR* use the `toolset` configure preset to specify the `ClangCL` toolset like so:
+  
+```json
+"cacheVariables": {
+  "CMAKE_BUILD_TYPE": "Debug",
+  "CMAKE_INSTALL_PREFIX": "${sourceDir}/out/install/${presetName}",
+},
+"toolset": "ClangCL",
```

Comment:
This assumes the current `generator` supports specifying toolsets (see [docs](https://cmake.org/cmake/help/latest/variable/CMAKE_GENERATOR_TOOLSET.html) for more info). For the `Ninja` generator this preset is invalid.

Reply:
You're right. I have pushed up a new commit to clarify that not all generators support this preset.

Reply:
Thanks for updating that. However I don't think ClangCl will be valid until the `Visual Studio 16 2019` generator, since LLVM was added in [version 16.1](https://devblogs.microsoft.com/cppblog/clang-llvm-support-in-visual-studio/). Also I'd be more explicit that this is only for Visual Studio generators. perhaps

> if you are using either `Visual Studio 16 2019` or `Visual Studio 17 2022` as your generator, you can use the `toolset` property in the configure preset to specify the `ClangCL` toolset:

Reply:
Done